cachito/web/api_v1.py

```diff
@@ -651,6 +653,14 @@ def get_request_logs(request_id):
     )
 
 
+def create_jsonify_response(content_manifest: Dict[str, Any]) -> flask.Response:
+    """Customize the dumped content manifest by sorting the keys."""
+    return current_app.response_class(
+        json.dumps(content_manifest, sort_keys=True),
+        mimetype=current_app.config["JSONIFY_MIMETYPE"],
+    )
```

Comment:
This would also likely be slower, it's memory vs. performance. Just wondering if you thought about this.

Reply:
One concern is there would be extra effort the manage the generated intermediate file. I'll rethink of it.

Reply:
@chmeliik 

When we use the `send_file`, one thing is important for us that is to cleanup the intermediate icm json files, so that such files will not occupy too much file system resources and become another kind of cost we have to monitor.

For example, based on the rox.git repo, the generated icm json file has 25M. Currently, I don't have the data of what is the percentage of requests have big icm data (>=1M or >=10M), but only get some basic data like there were around 500 HTTP GET requests in the past 24 hours and this endpoint is called frequently almost every day. Therefore, a periodic task is required to run, probably once a day, to execute the cleanup.

On the other hand, is it doable to serve the `/content-manifest` and `/download` endpoints by specific pods, separate from the api pods? If yes, we can adjust resources strategically among the pods.

Any idea, thought or comments? Thanks. :)

Reply:
I was thinking the files could be temporary, something like this
```python
with tempfile.TemporaryFile() as f:
    json.dump(content_manifest, f)
    return flask.send_file(f)
```
This would hurt the performance, since writing a file is guaranteed to be slower than generating the string in memory, but might help memory usage. Probably not worth doing at the moment.

Reply:
Let's do it. Please have a look at the new commit. :)